tests/next_tests/definitions.py

```diff
@@ -148,7 +151,7 @@ class ProgramFormatterId(_PythonObjectIdMixin, str, enum.Enum):
     (USES_ZERO_DIMENSIONAL_FIELDS, XFAIL, UNSUPPORTED_MESSAGE),
     (STARTS_FROM_GTIR_PROGRAM, SKIP, UNSUPPORTED_MESSAGE),
 ]
-GTIR_DACE_SKIP_TEST_LIST = [
+GTIR_DACE_SKIP_TEST_LIST = DOMAIN_INFERENCE_SKIP_LIST + [
     (USES_DYNAMIC_OFFSETS, XFAIL, UNSUPPORTED_MESSAGE),
```

Comment:
In order to see if DaCe GTIR backend supports dynamic offsets, we would have to delete line 155 (or I can wait for next merge commit, after #1648 is merged.

Reply:
I tested it locally and it works.
<img width="750" alt="image" src="https://github.com/user-attachments/assets/fb4e6cb8-abc9-4d86-bf71-edd6025a37d3">